features/step_definitions/documents-steps.js

```diff
@@ -122,6 +122,21 @@ Then(/The document "(.*?)" should( not)? exist/, async function (id, not) {
   }
 });
 
+Then(/The document "(.*?)" should( not)? multiple exist:$/, async function (id, not) {
```

Comment:
If you are adding a step using `mExists` then the step should be able to check multiple documents existence.  

Otherwise you can also use the [generic step to execute and API action](https://github.com/kuzzleio/kuzzle/blob/master/features/step_definitions/controllers-steps.js#L30) and then check the result with [another generic step checking array result](https://github.com/kuzzleio/kuzzle/blob/master/features/step_definitions/controllers-steps.js#L64